.github/dependabot.yml

```diff
@@ -7,10 +7,22 @@ updates:
     day: "monday"
     time: "09:00"
     timezone: "Europe/Berlin"
+  commit-message:
+    prefix: "ci"
+  labels:
+    - run-ci
+  reviewers:
+    - "cloudfoundry/wg-app-runtime-platform-networking-extensions-approvers"
```

Comment:
Since we have code-owners set up this shouldn't be needed?

Reply:
yes. removed.